src/pages/projects.jsx

```diff
@@ -59,6 +60,48 @@ const ProjectImage = styled.img`
     max-width: 100%;
 `;
 
+const Player1 = ({ classCircles }) => (
+    <ReactPlayer
+        url={Circles}
+        className={classCircles}
+        width="100%"
+        height="100%"
+        controls={false}
+        muted
+        playing
+        loop
+    />
+);
+
+const CirclesVideo = styled(Player1)`
+    margin: 0 auto 100px auto;
+    display: block;
+    width: auto;
+    height: auto;
+    max-width: 100%;
+`;
+
+const Player2 = ({ classCityscape }) => (

```

Comment:
Give this a more specific name. Instead of `Player2`, you can call this `CityscapeVideo`. (The styled component you currently have as `CityscapeVideo` should be renamed based on  [this comment](https://github.com/dollarbeale/website_project/pull/41/files#r1127182749))